src/lib.rs

```diff
@@ -766,6 +731,13 @@ where
     Ok(Deserialize::deserialize(deserializer).unwrap_or_default())
 }
 
+pub fn toml_from_slice<T>(s: &'_ [u8]) -> Result<T, Error>
```

Comment:
Better to keep this private:
```suggestion
fn toml_from_slice<T>(s: &'_ [u8]) -> Result<T, Error>
```

Reply:
Ah right, wasn't thinking.